packages/package_info_plus/package_info_plus_linux/pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
   flutter: ">=1.20.0"
 
 dependencies:
-  package_info_plus_platform_interface: ^1.0.2
+  package_info_plus_platform_interface: ^2.0.0
```

Comment:
I am confident this is going to cause an error.

The `melos version` is giving `package_info_plus_linux` a minor version bump (from 1.0.5 to 1.0.6), but the dependencies are updated to `package_info_plus_platform_interface: ^2.0.0` which is a major version.

Reply:
The problem will be that someone downloading `package_info_plus` `^1.0.0` is going to download `package_info_plus_linux 1.0.6` which depends on `package_info_plus_platform_interface 2.0.0`, but at the same time,  `package_info_plus ^1.0.0` depends on `package_info_plus_platform_interface ^1.0.0`